source/reference/options.txt

```diff
@@ -25,19 +25,22 @@ General Options
 
 .. option:: --eval <javascript>
 
-   Evaluates a JavaScript expression that is specified as an argument.
-   ``mongosh`` does not load its own environment when evaluating
-   code. As a result many options of the shell environment are not
-   available.
-
-   The result of evaluating a JavaScript argument is printed to your 
-   command line.
+   Evaluates a JavaScript expression. You can use a single ``--eval``
+   argument or multiple ``--eval`` arguments together.
+   
+   After ``mongosh`` evaluates the ``--eval`` argument, it prints the
+   results to your command line. If you use multiple ``--eval``
+   statements, ``mongosh`` only prints the results of the last
+   ``--eval``.
 
-   To get output suitable for automated parsing, use
-   ``EJSON.stringify()``.
+   **Example: Format Output**
```

Comment:
[optional suggestion] 

Consider using the "Example" admonition for these two cases.

Reply:
Thanks @jmd-mongo

This might be a good spot for it, but I think we agreed not to use the "example" since it can be distracting on the page, especially in this case where the existing example is quite long.